src/LinAlg/hiopVectorRaja.hpp

```diff
@@ -275,7 +279,8 @@ class hiopVectorRajaPar : public hiopVector
   virtual bool isinf_local() const;
   virtual bool isfinite_local() const;
   
-  virtual void print(FILE* file=nullptr, const char* message=nullptr,int max_elems=-1, int rank=-1) const;
+  virtual void print(FILE*, const char* withMessage=NULL, int max_elems=-1, int rank=-1) const;
```

Comment:
I assume the default values are initialized in the `.cpp` file instead...

Reply:
updated to code to default to `stdout` when the output file is null.